pkg/services/rpcsrv/server.go

```diff
@@ -900,6 +901,12 @@ func (s *Server) getPeers(_ params.Params) (any, *neorpc.Error) {
 	return peers, nil
 }
 
+func (s *Server) getPeerHeights(_ params.Params) (any, *neorpc.Error) {
```

Comment:
Could you please update the RPC client? https://github.com/nspcc-dev/neo-go/blob/master/pkg/rpcclient/rpc.go